libcentrifugo/application.go

```diff
@@ -498,7 +498,7 @@ func (app *Application) pubPing() error {
 		Goroutines: runtime.NumGoroutine(),
 		NumCPU:     runtime.NumCPU(),
 		Gomaxprocs: runtime.GOMAXPROCS(-1),
-		Metrics:    *app.metrics,
+		Metrics:    *app.metrics.GetSnapshot(),
```

Comment:
I tried to reproduce in various ways, with concurrent API calls, fast snapshot update interval etc - works fine. Only race detector showed lots of race warnings.